src/issue.ts

```diff
@@ -52,92 +52,101 @@ async function getRepoIssues(repo: Repo, flags?: Flags): Promise<IssueResult> {
   const result = {issues: new Array<Issue>(), repo};
   let res!: GaxiosResponse<IssuesApiResponse>;
   const rootUrl = 'https://drghs.endpoints.devrel-prod.cloud.goog/api/v1';
-  const url = `${rootUrl}/${repo.repo}/issues?key=${apiKey}&closed=false`;
-  try {
-    res = await request<IssuesApiResponse>({url});
-  } catch (e) {
-    console.warn(`Error fetching issues for ${repo.repo}.`);
-    //console.warn(e);
-    return result;
-  }
 
-  if (!res.data || !res.data.issues) {
-    return result;
-  }
+  let pageToken = '';
+  while (pageToken === '' && result.issues.length < 1) {
+    const url = `${rootUrl}/${repo.repo}/issues?key=${apiKey}&closed=false${
```

Comment:
I find statements like this inside of string interpolation harder to read.  Could I trouble you to break this out into a variable above?